### DISCUSSION TITLE
Willkommen im luca-GitHub

### DISCUSSION THREAD

**Discussion by philipp-berger** [lucaapp/security-concept#3] [General]
Hier haben wir unser lebendiges Sicherheitskonzept veröffentlicht. Wir freuen uns auf konstruktives Feedback, spannende Fragen und Diskussionen mit euch. Bitte bleibt respektvoll dabei. Das luca-System hat den Anspruch immer besser und sicherer zu werden, deshalb arbeiten wir kontinuierlich daran. Wir freuen uns, wenn ihr ein Teil dieser Aufgabe werdet.
Euer luca-Team

**Comment by Boerge64** [comment]
Was ich sehr vermisse, ist eine vernüpftige Hilfefunktion! Und in dem Zusammenhang, eine Mailadresse für Nachfragen. Deshalb benutze ich diesen Platz - auch wie von mir vermutet, dafür nicht vorgesehen - hier für meine Anfrage:
Wie funktioniert Auschecken??? Der Button Ausschecken funktioniert schon mal nicht. Eine Hilfestellung im Netz fehlanzeige. Im Gegenteil, ich habe mehrere mit dem selben Problem gefunden. Nun bin ich schon Tagelang an einem Ort "Eingescheckt", an dem ich überhaupt nicht mehr bin und habe keine Möglichkeit das zu ändern.

**Reply by anette-03** [reply]
Möglicherweise das gleiche Problem, wie bei mir. Es lag aber daran, dass ich den Button zum manuellen Auschecken immer geklickt (berührt) habe und man eine Wischbewegung von rechts nach links machen muss, um sich auszuchecken (das deutet der Pfeil innerhalb des Buttons an). Hier wäre eine Überarbeitung des Designs des Buttons hilfreich um es eindeutiger zu machen ;)
